dallinger/recruiters.py

```diff
@@ -39,17 +39,39 @@
 logger = logging.getLogger(__file__)
 
 
-def _get_queue(name="default"):
-    # Connect to Redis Queue
-    return Queue(name, connection=redis_conn)
-
-
 # These are constants because other components may listen for these
 # messages in logs:
 NEW_RECRUIT_LOG_PREFIX = "New participant requested:"
 CLOSE_RECRUITMENT_LOG_PREFIX = "Close recruitment."
 
 
+def run_status_check():
+    """Update participant status via all active recruiters.
+
+    Queries database for all working participants and asks owning recruiter to
+    verify that the status we have for them matches the status on the
+    recruitment platform.
+
+    If a recruiter finds discrepancies, it will enqueue a command to correct the
+    status for each participant with a problem.
+    """
+    # from dallinger import experiment
+
+    # experiment.load()
+    participants_by_recruiter_nick = defaultdict(list)
+    for participant in Participant.query.all():
+        participants_by_recruiter_nick[participant.recruiter_id].append(participant)
+
+    logger.warning(
```

Comment:
Now that you have fixed the logging in background processes, can this be changed to `logger.info`?

Reply:
Done